package/MDAnalysis/coordinates/memory.py

```diff
@@ -286,6 +286,29 @@ def __init__(self, coordinate_array, order='fac',
         self.ts.time = -1
         self._read_next_timestep()
 
+    @staticmethod
+    def parse_n_atoms(filename, order='fac', **kwargs):
+        """Deduce number of atoms in a given array of coordinates
+
+        Parameters
+        ----------
+        filename : numpy.ndarray
```

Comment:
The use of the `filename` parameter here seems a bit confusing given that it actually refers to a numpy array object, but perhaps that is simply because I've not made much use of the memoryreader stuff myself yet??

Reply:
yeah so memoryreader abuses the API to use its sole argument as a numpy array not a filename